dataactbroker/handlers/submission_handler.py

```diff
@@ -136,6 +137,12 @@ def get_submission_metadata(submission):
     test_sub = find_existing_submissions_in_period(submission.cgac_code, submission.frec_code,
```

Comment:
Would it make more sense to refactor `find_existing_submissions_in_period` to return the list of submissions (`[]` if not found) and then have another function to JSONify the response for `check_year_and_quarter`? Totally a matter of preference.